Speckle_Adapter/Diffing/Diffing.cs

```diff
@@ -0,0 +1,51 @@
+﻿using BH.oM.Base;
+using BH.oM.DataStructure;
+using BH.Adapter.Speckle.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using System.Collections;
+using SpeckleCore;
+
+namespace BH.Adapter.Speckle
+{
+    public partial class SpeckleAdapter
+    {
+
+        private bool DiffingByBHoMGuid(List<IObject> objectsToBePushed, out List<IObject> objectsCreated)
+        {
+            // This part is actually useless -- written assuming IBHoMObjects had static GUID (instead they are re-instantiated at any modification).
```

Comment:
If this is not used anymore, we can remove it altogether - we have the git history if we want to retrieve it!

Reply:
No it's used -- comment misleading.